interface/super/edit_list.php

```diff
@@ -1238,97 +1230,104 @@ function lister() {
+            <th><?php echo xlt('Color'); ?></th>
+            <th><?php echo xlt('Alert Time'); ?></th>
+            <th><?php echo xlt('Check In'); ?>&nbsp;&nbsp;&nbsp;&nbsp;</th>
+            <th><?php echo xlt('Check Out'); ?></th>
+            <th><?php echo xlt('Code(s)'); ?></th>
         <?php elseif ($list_id == 'issue_types') : ?>
-            <th class="font-weight-bold"><?php echo xlt('OpenEMR Application Category'); ?></th>
-            <th class="font-weight-bold"><?php echo xlt('Active{{Issue}}'); ?></th>
-            <th class="font-weight-bold"><?php echo xlt('Order{{Sequence}}'); ?></th>
-            <th class="font-weight-bold"><?php echo xlt('Type'); ?></th>
-            <th class="font-weight-bold"><?php echo xlt('Plural'); ?></th>
+            <th><?php echo xlt('OpenEMR Application Category'); ?></th>
+            <th><?php echo xlt('Active{{Issue}}'); ?></th>
+            <th><?php echo xlt('Order{{Sequence}}'); ?></th>
+            <th><?php echo xlt('Type'); ?></th>
+            <th><?php echo xlt('Plural'); ?></th>
             <?php //show translation column if not english and the translation lists flag is set
             if ($GLOBALS['translate_lists'] && $_SESSION['language_choice'] > 1) {
                 echo "<th class='font-weight-bold'>" . xlt('Translation') . "<span class='help' title='" . xla('The translated Title that will appear in current language') . "'> (?)</span></th>";
             } ?>
-            <th class="font-weight-bold"><?php echo xlt('Singular'); ?></th>
+            <th><?php echo xlt('Singular'); ?></th>
             <?php //show translation column if not english and the translation lists flag is set
             if ($GLOBALS['translate_lists'] && $_SESSION['language_choice'] > 1) {
                 echo "<th class='font-weight-bold'>" . xlt('Translation') . "<span class='help' title='" . xla('The translated Title that will appear in current language') . "'> (?)</span></th>";
             } ?>
-            <th class="font-weight-bold"><?php echo xlt('Mini'); ?></th>
+            <th><?php echo xlt('Mini'); ?></th>
             <?php //show translation column if not english and the translation lists flag is set
             if ($GLOBALS['translate_lists'] && $_SESSION['language_choice'] > 1) {
                 echo "<th class='font-weight-bold'>" . xlt('Translation') . "<span class='help' title='" . xla('The translated Title that will appear in current language') . "'> (?)</span></th>";
             } ?>
-            <th class="font-weight-bold"><?php echo xlt('Style'); ?></th>
-            <th class="font-weight-bold"><?php echo xlt('Force Show'); ?></th>
-            <th class="font-weight-bold"><?php echo xlt('Access Control'); ?></th>
+            <th><?php echo xlt('Style'); ?></th>
+            <th><?php echo xlt('Force Show'); ?></th>
+            <th><?php echo xlt('Access Control'); ?></th>
+        <?php elseif ($list_id == "external_patient_education"): ?>
```

Comment:
Would just place this if below where the Notes label is then only need to use 'URL' instead of 'Notes' at that place (and no need need to touch other colums). Then all the other columns will function like standard lists do (such as translations do) and then it won't break if any changes are made to standard columns.
(also note should use th and not td in initial tag)